packages/apps/patterns/react-composer/src/styles/markdown.ts

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { mx } from '@dxos/react-components';
+
+export type HeadingLevel = 1 | 2 | 3 | 4 | 5 | 6;
+
+export const heading: Record<HeadingLevel, string> = {
```

Comment:
related to question about colors above -- re handling custom themes (colors, sizes) for a possible presentation view. this may not be necessary since in my demo i just scale the container so the geometry didn't change.

Reply:
The sizes are currently available in our Tailwind config, so if you need to use those at runtime they can be retrieved.